packages/tsconfig.build.json

```diff
@@ -0,0 +1,9 @@
+{
+  "files": [],
+  "references": [
+		{ "path": "api" },
+		// "error TS5053: Option 'noEmit' cannot be specified with option 'composite'."
+		// TS incremental build is not playing well with Babel projects
+    // { "path": "web" }
+  ]
+}
```

Comment:
This is possible in TS version 4: https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/#noemit-and-incremental.

microsoft/TypeScript#39122